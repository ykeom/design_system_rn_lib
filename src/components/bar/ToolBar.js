import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class ToolBar extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>tool bar</Text>
      </View>
    );
  }
}

export default ToolBar;
