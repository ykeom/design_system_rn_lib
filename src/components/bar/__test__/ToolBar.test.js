import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ToolBar from '../ToolBar';

describe('ToolBar', () => {
  it('should render', () => {
    const component = shallow(<ToolBar />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
