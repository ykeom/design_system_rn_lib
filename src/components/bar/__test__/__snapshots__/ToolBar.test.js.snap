// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToolBar should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    tool bar
  </Text>
</View>
`;
